### PR TITLE
Added linting to travis build

### LINKED ISSUE #321: Add linting to travis CI build

### PR DESCRIPTION
**Pull request by mamaso**
resolves #321

### CODE REVIEW

**Comment by mamaso** [comment]
resolves #322

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ node_modules
 
 jsdoc/out
 azureMobile.js
+test/config.js
```
Comment (danderson00) [inline .gitignore]: This should not be necessary. Tests no longer use config.js, they use azureMobile.js.

**Comment by danderson00** [comment]
Nice, really simple.

File: .gitignore (hunk 1 above)
Comment (mamaso) [inline .gitignore]: Yep, knew this had happened, thought it was azureMobile -> config.js and not the other way around

**Pull request merged**